api/src/main/java/io/grpc/CompositeChannelCredentials.java

```diff
@@ -46,4 +46,9 @@ public ChannelCredentials getChannelCredentials() {
   public CallCredentials getCallCredentials() {
     return callCredentials;
   }
+
+  @Override
+  public ChannelCredentials withoutBearerTokens() {
+    return channelCredentials.withoutBearerTokens();
```

Comment:
Should we just return `this` instead?

Reply:
CallCredentials will need to be removed recursively so this is correct. Note the channelCredentials here itself could be CompositeChannelCredentials.